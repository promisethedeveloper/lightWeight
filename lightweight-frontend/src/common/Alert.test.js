import React from "react";
import { render } from "@testing-library/react";
import Alert from "./Alert";

/** Smoke test */
it("should render without crashing", () => {
	render(<Alert />);
});

/** Snapshot test */
it("should match snapshots", () => {
	const { asFragment } = render(<Alert />);
	expect(asFragment()).toMatchSnapshot();
});

it("should show a 'Not found' error message", () => {
	const { queryByText } = render(
		<Alert type="danger" messages={["Not found"]} />
	);
	expect(queryByText("Not found")).toBeInTheDocument();
});
