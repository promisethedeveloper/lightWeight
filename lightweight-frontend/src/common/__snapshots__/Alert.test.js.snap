// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshots 1`] = `
<DocumentFragment>
  <div
    class="alert alert-danger"
    role="alert"
  />
</DocumentFragment>
`;
